docs/integrations/fetching-incidents.md

```diff
@@ -145,6 +145,30 @@ If there are no incidents to return,  *demisto.incidents()* returns an empty lis
 demisto.incidents([])
 ```
 
+## Fetch History
+In XSOAR Versions 6.8 and above, it is possible to observe the results of the last **fetch-incidents**/**fetch-indicators** runs using the Fetch History modal. For more details, visit the [Fetch History](https://xsoar.pan.dev/docs/reference/articles/troubleshooting-guide#fetch-history) documentation.
+
+When implementing a **fetch-incidents** command, in some cases we can populate extra data for the following columns in the modal:
+
+1. In Long-Running Integrations, the info/error message given in `demisto.updateModuleHealth()` will be displayed in the **Message** column. Use the *is_error* boolean argument of this method to determine the message type. For example:
+   
+   ```python
+   demisto.updateModuleHealth("Could not connect to client.", is_error=True)
+   ```
+   The above line will produce a new record in the modal, and its **Message** value will be `Error: Could not connect to client.`.
+
+1. If Incidents on the 3rd-party product have IDs, it is possible to display them in the **Source IDs** column by adding the `dbotMirrorId` field as part of the incident dictionary. For example:
+   
+   ```python
+   demisto.incidents([
+       {"name": "This is an incident.", "dbotMirrorId": "123"},
+       {"name": "This is another incident.", "dbotMirrorId": "124"},
+   ])
+   ```
+   The above will produce a new record in the modal, and its **Source IDs** value will be `123, 124`.
+   
+   **Note:** the population of the fetch run information occurs **before** the classification, therefore this field must be defined at the integration code level rather than the classification & mapping level.
```

Comment:
```suggestion
   **Note:** the population of the fetch history information occurs **before** the classification, therefore this field must be defined at the integration code level rather than the classification & mapping level.
```